src/qt/glcanvas.cpp

```diff
@@ -505,14 +539,16 @@ bool wxGLCanvas::ConvertWXAttrsToQtGL(const int *wxattrs, QGLFormat &format)
 
             case WX_GL_SAMPLE_BUFFERS:
                 format.setSampleBuffers(v);
-                // can we somehow indicate if it's not supported?
```

Comment:
Not sure why these comments were removed, it doesn't look like the problem they concern was addressed, i.e. we still don't seem to check whether the value is actually supported.

Reply:
Not sure where these comments came from, maybe original qt port, where some options were not supported, and the case statement was a simple break.  Anyway, the code supports setting the options indicated now.  So the comments no longer mean much.